Content/default/tests/Client/Client.Tests.fs

```diff
@@ -12,19 +13,28 @@ let client =
             let newTodo = Todo.create "new todo"
             let model, _ = init ()
 
-            let model, _ = update (AddedTodo newTodo) model
+            let model, _ = update (SaveTodo(Finished newTodo)) model
 
-            Expect.equal model.Todos.Length 1 "There should be 1 todo"
-            Expect.equal model.Todos.[0] newTodo "Todo should equal new todo"
+            Expect.equal
+                (model.Todos |> RemoteData.map _.Length |> RemoteData.defaultValue 0)
+                1
+                "There should be 1 todo"
+
+            Expect.equal
+                (model.Todos
+                 |> RemoteData.map List.head
+                 |> RemoteData.defaultValue (Todo.create ""))
+                newTodo
+                "Todo should equal new todo"
     ]
 
 let all =
     testList "All" [
-//-:cnd:noEmit
+        //-:cnd:noEmit
```

Comment:
Don't indent these comments. It breaks the tests. See https://github.com/SAFE-Stack/SAFE-template/pull/581/files.

Perhaps it's worth a regular comment to say that?

Reply:
This was done by Fantomas :-( Maybe we need to raise an issue for that?

Reply:
cc @nojaf